controllers/hibp.js

```diff
@@ -13,6 +13,10 @@ const log = mozlog("controllers.hibp");
 
 
 async function notify (req, res) {
+  if (!req.token || req.token !== AppConstants.HIBP_NOTIFY_TOKEN) {
+    const errorMessage = "HIBP notify endpoint requires valid authorization token.";
```

Comment:
Another small, useless nit... but not sure if we should export this value, since we seem to use it 2-3x below in ./tests/controllers/hibp.test.js